app/src/main/java/com/example/dazn_movie_player/MainActivity.kt

```diff
@@ -13,14 +14,19 @@ import retrofit2.Response
 
 
 class MainActivity : AppCompatActivity() {
+
+    lateinit var mListRecyclerView: RecyclerView;
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        rv_movies_list.layoutManager = LinearLayoutManager(this)
-        rv_movies_list.setHasFixedSize(true)
+
+
+        mListRecyclerView=findViewById<RecyclerView>(R.id.rv_movies_list)
+        mListRecyclerView.layoutManager = LinearLayoutManager(this)
+        mListRecyclerView.setHasFixedSize(true)
         getMovieData { movies : List<Movie> ->
-            rv_movies_list.adapter = MovieAdapter(movies)
```

Comment:
callback(response.body()!!.movies) ,Please do null check condition instead using not null assertion to avoid exceptions